clusterloader2/pkg/test/simpletestexecutor.go

```diff
@@ -152,12 +153,42 @@ func (ste *simpleTestExecutor) ExecutePhase(ctx Context, phase *api.Phase) []err
 
 // ExecuteObject executes single test object operation based on provided object configuration.
 func (ste *simpleTestExecutor) ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32) []error {
+	var errList []error
```

Comment:
I would suggest here the approach of passing another parameter to this function for deciding create/update/delete (since you're anyway already calculating those cases in `ExecutePhase`). It'll make this function much simpler and stateless I hope.

Reply:
good idea. done.